whois-db/src/main/resources/applicationContext-whois-base.xml

```diff
@@ -11,8 +11,7 @@
     <context:mbean-export/>
 
     <context:annotation-config />
-    <bean class="net.ripe.db.whois.scheduler.SchedulerConfig" />
-
     <import resource="applicationContext-nrtm.xml"/>
+    <bean class="net.ripe.db.whois.scheduler.SchedulerConfig" />
```

Comment:
You didn't like it being on line 14? ;)

Reply:
haha...while trying to figure out what could be going wrong...I moved the configurations around a bit to match whois-internal :)

Reply:
Ok :)